LFS171x/fabric-material/tuna-app/registerAdmin.js

```diff
@@ -4,6 +4,9 @@
 */
 /*
  * Chaincode Invoke
+
+Code contributed by the Hyperledger Team 
+  Original code can be found here: https://gerrit.hyperledger.org/r/#/c/14395/4/fabcar/enrollAdmin.js
```

Comment:
This one references gerrit, but the other Fabric once reference github. Why the difference?

Reply:
the registerAdmin.js code is not reflected on github yet.